sunpycube/cube/NDCube.py

```diff
@@ -10,6 +11,7 @@
 import astropy.nddata
 import numpy as np
 import copy
+import warnings
```

Comment:
```to_sunpy()``` should raise an error, not a warning (see comment below).  So this import can be removed.